src/plugins/safeSnap/utils/umaModule.ts

```diff
@@ -422,53 +447,11 @@ export const getModuleDetailsUmaGql = async (
     assertionId !==
     '0x0000000000000000000000000000000000000000000000000000000000000000';
 
-  // Search for requests with matching ancillary data
-  const oracleContract = new Contract(
-    optimisticOracle,
-    UMA_ORACLE_ABI,
-    provider
-  );
-
-  const latestBlock = await provider.getBlock('latest');
-  // modify this per chain. this should be updated with constants for all chains. start block is og deploy block.
-  // this needs to be optimized to reduce loading time, currently takes a long time to parse 3k blocks at a time.
-  const oGstartBlock = network === '1' ? 17167414 : 0;
-  const maxRange = network === '1' ? 3000 : 10000;
-
-  // TODO: use og subgraph to replace need for these calls
-  const [transactionsProposedEvents, executionEvents] = await Promise.all([
-    // Check if this specific proposal has already been executed.
-    // note usage of pageEvents, which query only based on a limit number of blocks within a broader range
-    // this prevents block range too large errors.
-    pageEvents(
-      oGstartBlock,
-      latestBlock.number,
-      maxRange,
-      ({ start, end }: { start: number; end: number }) => {
-        return moduleContract.queryFilter(
-          moduleContract.filters.TransactionsProposed(),
-          start,
-          end
-        );
-      }
-    ),
-    pageEvents(
-      oGstartBlock,
-      latestBlock.number,
-      maxRange,
-      ({ start, end }: { start: number; end: number }) => {
-        return moduleContract.queryFilter(
-          moduleContract.filters.ProposalExecuted(proposalHash),
-          start,
-          end
-        );
-      }
-    )
-  ]);
```

Comment:
we can remove all events now from the graph function call now that we have OG subgraphs